modular_skyrat/master_files/code/modules/client/preferences/antag_opt_in_preferences.dm

```diff
@@ -0,0 +1,91 @@
+/datum/config_entry/flag/disable_antag_opt_in_preferences
```

Comment:
btw if you make a new module youll probably want to move this file into the module as well